File: C99/1000-1099/1020-1029/1020.c
#include <stdio.h>

int main()
{
    int n, ano = 0, mes = 0;
    scanf("%d", &n);
    if (n >= 365)
    {
        ano = n / 365;
        n %= 365;
    }
    if (n >= 30)
    {
        mes = n / 30;
        n %= 30;
    }
    printf("%d ano(s)\n", ano);
    printf("%d mes(es)\n", mes);
    printf("%d dia(s)\n", n);
    return 0;
}

File: beecrowd/1000-1099/1020-1029/1020.py
days, months, years = int(input()), 0, 0
if days >= 365:
    years = int(days / 365)
    days %= 365
if days >= 30:
    months = int(days / 30)
    days %= 30
print(f"{years} ano(s)")
print(f"{months} mes(es)")
print(f"{days} dia(s)")